parsl/tests/test_error_handling/test_htex_worker_failure.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+import datetime
+import time
+
+import pytest
+
+import parsl
+from parsl.app.app import python_app
+from parsl.executors.high_throughput.process_worker_pool import WorkerLost
+
+def local_setup():
+    from parsl.tests.configs.htex_local import config
```

Comment:
Don't mutate global config in test - use fresh_config pattern so these changes don't affect other tests

Reply:
Fixed